src/polymer/polymer-element.ts

```diff
@@ -83,6 +83,7 @@ export class ScannedPolymerElement extends ScannedElement {
   //     level. Remove them here, they should only exist on PolymerElement.
   domModule?: dom5.Node;
   scriptElement?: dom5.Node;
+  slots: Slot[];
```

Comment:
Ideally `slots` would live on ScannedElement and Element, as it's not a polymer specific thing (even if we only know how to recognize slots on polymer elements for the moment)